azure-pipelines-official.yml

```diff
@@ -44,6 +44,10 @@ parameters:
   type: boolean
   default: true
 
+- name: VisualStudioCherryPickSHA
```

Comment:
Not clear what this does ... can you add a comment describing what this parameter is used for?

Reply:
Oh ... so this lets us pick a commit of VS that we want to run against? Basically means we can retest a change against the same VS baseline?

Reply:
not exactly - it allows us to cherry-pick a commit from another VS branch onto our insertion branch.  For example if we need to do a dual insertion, or a roslyn change requires another VS-side change.  Instead of having to wait for the PR to be created, then cherry-pick it over, the roslyn insertion tool can do it when it creates the insertion